internal/website/gatherexamples/gatherexamples.go

```diff
@@ -213,10 +229,10 @@ rewrite:
 			sb.WriteByte('\n')
 			continue
 		}
-		const importBlankPrefix = "// import _ "
+		const importBlankPrefix = pragmaPrefix + "Add a blank import: _ "
 		indent, lineContent := line[:start], line[start:]
 		switch {
-		case lineContent == "// Variables set up elsewhere:":
+		case lineContent == pragmaPrefix+"Skip until next blank line.":
```

Comment:
spaces around + for consistency please

Reply:
This is not my choice, it is how `gofmt` wants it. It's not obvious to me why it doesn't have spaces here, but does have spaces on the `const` line above.